fix for: ./configure: line 16872: syntax error near unexpected token `newline'
see also https://github.com/TGAC/KAT/issues/174 + https://github.com/gmarcais/Jellyfish/pull/178
--- KAT-Release-2.4.2/deps/jellyfish-2.2.0/configure.ac.in.orig	2018-05-04 10:55:58.000000000 +0200
+++ KAT-Release-2.4.2/deps/jellyfish-2.2.0/configure.ac.in	2023-03-13 17:55:27.316962287 +0100
@@ -19,9 +19,9 @@
 
 # Check for md5 or md5sum
 AC_ARG_VAR([MD5], [Path to md5 hashing program])
-AS_IF([test "x$MD5" = "x"], AC_CHECK_PROG([MD5], [md5sum], [md5sum]), [])
-AS_IF([test "x$MD5" = "x"], AC_CHECK_PROG([MD5], [md5], [md5 -r]), [])
-AS_IF([test "x$MD5" = "x"], AC_MSG_ERROR([Could not find md5 hashing program in your path]), [])
+AS_IF([test "x$MD5" = "x"], [AC_CHECK_PROG([MD5], [md5sum], [md5sum])], [])
+AS_IF([test "x$MD5" = "x"], [AC_CHECK_PROG([MD5], [md5], [md5 -r])], [])
+AS_IF([test "x$MD5" = "x"], [AC_MSG_ERROR([Could not find md5 hashing program in your path])], [])
 
 # Check for yaggo
 AC_ARG_VAR([YAGGO], [Yaggo switch parser generator])
